### ISSUE TITLE
[Feature request] Would it be possible to support the `--listTests` CLI flag?

### ISSUE THREAD

**Issue by benedfit** [storybookjs/test-runner#328]
I'd like to be able to use the new "Re-run only failed tests" feature in CircleCI. This entails being able to run specs on an individual basis, is this something `test-runner` could provide?

Here's how I am achieving this with Jest:

```
yarn jest --listTests | circleci tests run --command="xargs yarn jest --ci --runInBand"
```

`--listTests` provides a space/new-line delimited list of specs to run, which are then piped to the second command, allowing certain tests to be run individually

**Comment by MH4GF** [comment]
Can't you run with this? `yarn test-storybook -- --listTests`
ref: https://github.com/storybookjs/test-runner/issues/230#issuecomment-1379472607

**Comment by benedfit** [comment]
Unfortunately not, I get `error: unknown option '--listTests'`

**Comment by MH4GF** [comment]
oh, in my project, I am probably getting the expected results by doing the following(passing two `--`):

```
yarn test -- -- --listTests
yarn run v1.19.0
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
$ test-storybook --stories-json -- --listTests
/private/var/folders/xxx/yyy/components-foo.test.js
/private/var/folders/xxx/yyy/components-bar.test.js
...snip...
```

**Comment by yannbf** [comment]
Hey peeps, this is now available in v0.21.0!
